routes/test/authRoutes.spec.js

```diff
@@ -0,0 +1,84 @@
+const request = require('supertest');
+const express = require('express');
+const router = require('../authRoutes'); // Import your router file here
+const connectDb = require('../../__tests__/fixtures/db')
+const makeFakeUser = require('../../__tests__/fixtures/fakeUser')
+const mongoose = require('mongoose');
+const { encrypt } = require('../../helpers/Password');
+
+const app = express();
+app.use(express.json());
+app.use('/api', router); // Mount the router under '/api' path
+
+// Start the Express app on a specific port for testing
+const PORT = 5000; // Choose a port for testing
+const server = app.listen(PORT, () => {
+  console.log(`Express server is running on port ${PORT}`);
+});
+
+const fakeUser = makeFakeUser();
+
+describe('Login User route', () => {
+
+  let db; // Store the database connection
+
+  beforeAll(async () => {
+    db = await connectDb(); // Connect to the database
+
+    // Insert the fake user into the database
+    await db.collection('users').insertOne(fakeUser);
+  });
+
+  it('Returns error if user is not found', async () => {
+    const nonExistingUser = {
+      email: 'iDontExist@test.dk',
+      password: encrypt("12345678")
+    }
+
+    // Send a Post request to the login endpoint
+    const response = await request('http://localhost:5000')
```

Comment:
It'd probably be best not to hardcode the port, as the testing port is defined above.
Something like `` `http://localhost${PORT}` ``
(also applies in signupRoutes)